jupyterlab_ucaip/src/components/models/model_predictions.tsx

```diff
@@ -322,9 +306,24 @@ deploy_model(model_id='${this.props.model.id}',
     if (endpoints[0].deployedModelId !== 'None') {
       return (
         <div>
-          {this.props.model.modelType === 'TABLE'
-            ? this.testTablesModelComponent()
-            : this.testOtherModelComponent()}
+          <Grid container style={{ margin: '0px', width: '100%' }}>
+            {this.props.model.modelType === 'TABLE'
+              ? this.testTablesModelComponent()
+              : this.testOtherModelComponent()}
+            <Grid
+              item
+              xs={8}
+              style={{ paddingLeft: '16px', paddingRight: '16px' }}
+            >
+              <CodeComponent>
+                {`from jupyterlab_ucaip import predict
+
+instance = ${this.getInstanceString()}
```

Comment:
Why the odd formatting here?

Reply:
Adding indentation includes it in the string literal so the simplest way to work around this was to start the code at the beginning of the lines